<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>duplicate_main</class>
 <widget class="QWidget" name="duplicate_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Duplicate Extended</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>553</width>
        <height>387</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>386</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="translate_text">
            <property name="text">
             <string>Translate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="t_x">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="t_y">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="t_z">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="rotate_text">
            <property name="text">
             <string>Rotate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="r_x">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="r_y">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="r_z">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="scale">
            <property name="text">
             <string>Scale:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="s_x">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="s_y">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="s_z">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Append Suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="app_suffix"/>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>551</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="shared_input">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Shared input graph</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>331</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Duplicates:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="number_copies">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="copies_slider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>25</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>551</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="dupe_input">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Duplicate input graph</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>551</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>85</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Geometry Type:</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>160</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="copy_radial">
          <property name="text">
           <string>Copy</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="instance_radial">
          <property name="text">
           <string>Instance</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="repeat_first">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Repeat first duplicate's transform</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="accept_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
